ios/Runner.xcodeproj/project.pbxproj

```diff
@@ -542,6 +542,7 @@
 				ASSETCATALOG_COMPILER_APPICON_NAME = LaunchImage;
 				CLANG_ENABLE_MODULES = YES;
 				CURRENT_PROJECT_VERSION = "$(FLUTTER_BUILD_NUMBER)";
+				DEBUG_INFORMATION_FORMAT = "dwarf-with-dsym";
```

Comment:
Is this needed for crashlytics?

Reply:
Yes, this is to test if everything is fine in debug but crashlytics need to have these files to detect debug issues